diff --git a/CMakeLists.txt b/CMakeLists.txt
index b616409..165b395 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -92,21 +92,3 @@ target_link_libraries(${MOCK_TEST_EXE}
 )
 
 add_test(${MOCK_TEST_EXE} ${MOCK_TEST_EXE})
-
-# Integration tests that use the real operating system PC/SC service.
-
-set(INTEGRATION_TEST_EXE lib${PROJECT_NAME}-test-integration)
-
-add_executable(${INTEGRATION_TEST_EXE}
-  tests/common/selectcard.hpp
-  tests/common/verify.hpp
-  tests/integration/test-get-certificate.cpp
-  tests/integration/test-authenticate.cpp
-  tests/integration/test-signing.cpp
-)
-
-target_link_libraries(${INTEGRATION_TEST_EXE}
-  ${PROJECT_NAME}
-  pcsc
-  GTest::Main
-)
