--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,6 +64,7 @@ add_subdirectory(lib/libpcsc-cpp)
 
 # Common testing options.
 
+if(BUILD_TESTING)
 enable_testing()
 
 find_package(GTest REQUIRED)
@@ -94,3 +95,4 @@ target_link_libraries(${MOCK_TEST_EXE}
 )
 
 add_test(${MOCK_TEST_EXE} ${MOCK_TEST_EXE})
+endif()
--- a/lib/libpcsc-cpp/CMakeLists.txt	2025-11-18 23:57:07.834445148 +0200
+++ b/lib/libpcsc-cpp/CMakeLists.txt	2025-11-18 23:59:42.198610681 +0200
@@ -54,7 +54,7 @@
 endif()
 
 # Common testing options.
-
+if(BUILD_TESTING)
 enable_testing()
 
 find_package(GTest REQUIRED)
@@ -92,4 +92,5 @@
   pcsc
   GTest::Main
 )
+endif()
 target_link_libraries(${PROJECT_NAME} PUBLIC pcsc)
