--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,5 +83,6 @@ target_link_libraries(${MOCK_TEST_EXE}
   pcsc-mock
   GTest::Main
 )
+target_include_directories(${MOCK_TEST_EXE} PRIVATE ${PCSC_INCLUDE_DIRS})
 
 add_test(${MOCK_TEST_EXE} ${MOCK_TEST_EXE})
 
--- a/include/pcsc-cpp/comp_winscard.hpp
+++ b/include/pcsc-cpp/comp_winscard.hpp
@@ -24,13 +24,8 @@
 
 #include <cstdint>
 
-#ifdef __APPLE__
 #include <PCSC/wintypes.h>
 #include <PCSC/winscard.h>
-#else
-#include <winscard.h>
-#undef IGNORE
-#endif
 
 #ifdef __APPLE__
 #pragma pack(1)
