diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f5973a..c503b40 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,9 +1,5 @@
 cmake_minimum_required(VERSION 3.22)
 
-if(NOT EXISTS "${CMAKE_SOURCE_DIR}/lib/libelectronic-id/README.md")
-    message(FATAL_ERROR "libelectronic-id submodule directory empty, did you 'git clone --recursive'?")
-endif()
-
 if($ENV{BUILD_NUMBER})
     set(BUILD_NUMBER $ENV{BUILD_NUMBER})
 elseif($ENV{CI_PIPELINE_IID})
